src/SourceBuild/tarball/content/README.md

```diff
@@ -0,0 +1,160 @@
+# dotnet/dotnet - Home of the .NET VMR
+
+This repository is a **Virtual Monolithic Repository (VMR)**. It includes all the code needed to build the .NET SDK, mirrored from many atomic repos (like `dotnet/runtime`). It also includes [source-build](https://github.com/dotnet/source-build), our whole-product build system. The VMR is currently experimental.
+
+What this means:
+- **Monolithic** - a join of multiple individual repositories that make up the whole product, such as [dotnet/runtime](https://github.com/dotnet/runtime) or [dotnet/sdk](https://github.com/dotnet/sdk).
+- **Virtual** - a mirror (not replacement) of product repos where sources from those repositories are synchronized into.
+- **Experimental** - not to be depended on as we reserve the right to delete the current instance and create a new, different one in its stead. See [Limitations](#limitations).
+
+In the VMR, you can find:
+- source files of [each product repository](#list-of-components) which are mirrored inside of their respective directories under [`src/`](src/),
+- tooling that enables [building the whole .NET product from source](https://github.com/dotnet/source-build) on Linux platforms,
+- small customizations, in the form of patches, applied on top of the original code to make the build possible,
+- *[in future]* E2E tests for the whole .NET product.
+
+More in-depth documentation about the VMR can be found in [VMR Design And Operation](https://github.com/dotnet/arcade/blob/main/Documentation/UnifiedBuild/VMR-Design-And-Operation.md#layout).  
+See also [dotnet/source-build](https://github.com/dotnet/source-build) for more information about our whole-product source-build.
+
+## Limitations
+
+**This is a work-in-progress and an experiment.**
+There are considerable limitations to what is possible at the moment. For an extensive list of current limitations, please see [Temporary Mechanics](https://github.com/dotnet/arcade/blob/main/Documentation/UnifiedBuild/VMR-Design-And-Operation.md#temporary-mechanics).
+
+The VMR is expected to become non-experimental by .NET 8 Preview 1 (Februrary, 2023) 
+This means it won't be short-lived anymore and we won't be reserving the right to delete and re-create it anymore.
+Other limitations might apply until the .NET 9 timeframe.
+See the [Unified Build roadmap](https://github.com/dotnet/arcade/blob/main/Documentation/UnifiedBuild) for more details.
+
+### Supported platforms / builds
+
+.NET Source-Build is supported on the oldest available .NET SDK feature update for each major release, and on Linux only.
+For example, if .NET `6.0.1xx`, `6.0.2xx`, and `7.0.1xx` feature updates are available from [dotnet.microsoft.com](https://dotnet.microsoft.com/en-us/download/dotnet/6.0), Source-Build will support `6.0.1xx` and `7.0.1xx`.
```

Comment:
```suggestion
For example, if .NET `6.0.1xx`, `6.0.2xx`, and `7.0.1xx` feature updates are available from [dotnet.microsoft.com](https://dotnet.microsoft.com/en-us/download/dotnet/6.0), Source-Build will support `6.0.1xx` and `7.0.1xx`.

That being said, the VMR has only been created after .NET 7.0 had been released so we only plan to use it for supporting .NET 8.0 and further. This means that the VMR's main follows the main of the product repositories (see [Source Synchronization Process](https://github.com/dotnet/arcade/blob/main/Documentation/UnifiedBuild/VMR-Design-And-Operation.md#source-synchronization-process)). The VMR will also follow suite with the release branches for .NET 8 (e.g. `release/8.0.1xx-preview1` and similar).
```

Reply:
@MichaelSimons can you help me formulating this paragraph? This is related to the other open threads in this PR and I am wondering whether this doesn't deserve its own section describing what people can actually find in the VMR

Reply:
We have not explicitly talked about this but I always somehow thought about it that way. Only seems reasonable?

Reply:
I have rewritten a bit. Can you have a look?

Reply:
> Only seems reasonable to synchronize all release branches of installer into their respective ones in the VMR? How else would we do servicing?

Agreed - I wanted to make sure everyone is thinking the same.

> I have rewritten a bit. Can you have a look?

Done - Added comments inline.